rclpy/src/rclpy/node.cpp

```diff
@@ -179,15 +179,15 @@ Node::get_node_names_impl(bool get_enclaves)
 }
 
 py::list
-Node::get_node_names_and_namespaces()
+Node::get_names_and_namespaces()
```

Comment:
@ahcorde I'd suggest we keep `node` in the name here. It's not a reference to the object itself but to remote instances. Same below.